scripts/ironic-common.sh

```diff
@@ -83,16 +108,20 @@ run_ironic_dbsync()
     if [[ "${IRONIC_USE_MARIADB}" == "true" ]]; then
         # It's possible for the dbsync to fail if mariadb is not up yet, so
         # retry until success
-        until ironic-dbsync --config-file /etc/ironic/ironic.conf upgrade; do
+        until ironic-dbsync --config-file "${IRONIC_CONF_DIR}/ironic.conf" \
```

Comment:
This formatting looks awful, can we skip this.

Reply:
done